services/domains/domains.go

```diff
@@ -1,69 +1,31 @@
 package domains
 
 import (
-	"math"
-	"strings"
-
-	CoinType "github.com/trustwallet/blockatlas/coin"
 	"github.com/trustwallet/blockatlas/pkg/blockatlas"
 	"github.com/trustwallet/blockatlas/pkg/errors"
 	"github.com/trustwallet/blockatlas/platform"
 )
 
-// TLDMapping Mapping of name TLD's to coin where they are handled
-var TLDMapping = map[string]uint{
-	".eth":         CoinType.ETH,
-	".xyz":         CoinType.ETH,
-	".luxe":        CoinType.ETH,
-	".kred":        CoinType.ETH,
-	".zil":         CoinType.ZIL,
-	".crypto":      CoinType.ZIL,
-	"@trust":       CoinType.FIO,
-	"@trustwallet": CoinType.FIO,
-	"@binance":     CoinType.FIO,
-	"@fiomembers":  CoinType.FIO,
-	"@":            CoinType.FIO, // any FIO domain
-}
-
 func HandleLookup(name string, coins []uint64) ([]blockatlas.Resolved, error) {
-	// Assumption: format of the name can be decided (top-level-domain), and at most one naming service is tried
-	name = strings.ToLower(name)
-	tld, err := getTLD(name)
-	if err != nil {
-		return nil, errors.E(err, "name format not recognized", errors.Params{"name": name, "coins": coins})
-	}
-	id, ok := TLDMapping[tld]
-	if !ok {
-		// special handling for FIO, any fio domain
-		if len(tld) >= 2 && tld[0] == '@' {
-			tld = string("@")
-			id, ok = TLDMapping[tld]
-			if !ok {
-				return nil, errors.E("name not found", errors.Params{"name": name, "coins": coins, "tld": tld})
-			}
+	// Visit all providers, try lookup with all matching ones
+	// There must be at least one provider (normally ==1)
+	ret := []blockatlas.Resolved{}
```

Comment:
make(]blockatlas.Resolved, 0)

Reply:
that's the same

Reply:
We are using make for it as preferred way of slice initialization

Reply:
changed